DFE-Digital/sts-plan-technology-for-your-school
# fix: Handle unrecognised OIDC requests

## Overview

Addresses ticket [178380](https://dev.azure.com/dfe-ssp/s190-schools-technology-services/_workitems/edit/178380/)

Sets `SkipUnrecognizedRequests` to true in `OpenIdConnectOptions`


## How to review the PR

- Go to `/auth/cb`
- Check that the application no longer throws an exception
- If not authenticated, should redirect to DfE sign in

## Checklist

Delete any rows that do not apply to the PR.
